Research/Research/RSDTaskPath.swift

```diff
@@ -50,6 +50,10 @@ public final class RSDTaskPath : NSObject, NSCopying {
     /// and the app needs to track what the scheduled timing is for the task.
     public var scheduleIdentifier: String?
     
+    /// The user info is a pointer to task data that is required to run this task. This is a
+    /// pointer so it can be anything needed by the task.
```

Comment:
I'm confused about referring to this as a pointer. Was it adapted from Obj-C code? Does the comment need to be updated?

Reply:
No, originally it was an `Any`. Then I thought that the Apple pattern is `userInfo` as a dictionary, but really, I don't see why it needs to be. Basically, I am looking for a name for a thing that can hold a reference to a `clientData` object.